### ISSUE TITLE
enhancement : Propose to create regions from a set of latitudes and longitudes columns

### ISSUE THREAD

**Issue by yvanlebras** [earnaud/MetaShARK-v2#122]
Instead of describing each site of a "site" column by the corresponding lat and long, maybe we can propose to create "region(s)", where user give lat and long columns + if only one region, nothing (the name) / if several regions, the column with regions names so MetaShARK can create on area by region looking at max and min of lat and long by region.

**Comment by earnaud** [comment]
I see how some things can be done. I have two comments: 
- it is a bit of a work to allow in the same time the current feature and this new one, and requires to think how to design it correctly.
- imo, it would be preferrable to use the polygons (valid in EML) for a better precision

**Comment by yvanlebras** [comment]
I agree with you on both points. Maybe a comment on the first one : I think we can use the actual manner to do if the user has a column in his file with such ""big area / region(s)"" so using it as sites. Moreover, we can maybe ask to the user "do you want to use the entire dataset as an area/region ?" so in fact in metashark you can ""just"" create a 4 points area who will encapsulate all lat and long points of this datasets. What do you think about that ?
Concerning the second point, maybe we can propose a map so the user can selct polygon on a map ? What do you think about that ?For an example of exactely this kind of feature, you can have a look at the Wallace Shiny app https://github.com/wallaceEcoMod/wallace

**Comment by yvanlebras** [comment]
To update this enhancement, it appears to me that EMLAL is writing min and max of longitude / latitude attributes. So if the user can say (ticking a box ?) that we want use the longitude and latitude attribute to display/present the overall geographic coverage of the dataset, maybe there is a way to "just" take these min/max info from EAL saved informations (but apparently this is not in the metadata_templates files...)?

**Comment by earnaud** [comment]
Coming back on this issue while reworking the custom interface. 

I might offer things like:
- draw custom shapes and points on a map (and adjust them by hand afterwards)
- directly add adjustable geocov input
- get geocov input from data objects
This would require some work (especially in terms of ergonomy and legibility) but would end the dual method (columns vs. custom). 

I can also keep most of my work as is, and try to allow the user to access a preview of her geographic coverage and edit it (but on a second thought, this might be more work than the first solution).
